lib/std/zig/parse.zig

```diff
@@ -247,6 +247,15 @@ fn parseTopLevelDecl(arena: *Allocator, it: *TokenIterator, tree: *Tree) !?*Node
         fn_node.*.lib_name = lib_name;
         if (eatToken(it, .Semicolon)) |_| return node;
         if (try parseBlock(arena, it, tree)) |body_node| {
+            // Functions marked `extern` have no body
+            if (extern_export_inline_token) |token| {
+                if (tree.tokens.at(token).id == .Keyword_extern) {
+                    try tree.errors.push(AstError{
+                        .UnexpectedExternBody = AstError.UnexpectedExternBody{ .token = token },
+                    });
+                    return error.ParseError;
+                }
+            }
```

Comment:
I think we want this code, but not until after 0.6.0 is released:

https://github.com/ziglang/zig/blob/662996e4a8826f62f76bdafdb7ccbb1d52210c96/lib/std/zig/parser_test.zig#L89-L101

Reply:
Oh right, `extern` and its thousand different meanings heh